lib/arch/avx/codec.c

```diff
@@ -32,7 +31,12 @@ BASE64_DEC_FUNCTION(avx)
 {
 #ifdef __AVX__
 	#include "../generic/dec_head.c"
-	#include "../sse42/dec_loop.c"
+	#include "../ssse3/dec_loop.c"
+	#if BASE64_WORDSIZE == 32
+		#include "../generic/32/dec_loop.c"
+	#elif BASE64_WORDSIZE == 64
```

Comment:
Doesn't the presence of AVX pretty much guarantee a word size of 64 bytes? (No action needed on this comment.)